sky/provision/gcp/constants.py

```diff
@@ -106,6 +104,94 @@
     '--env LD_LIBRARY_PATH=/usr/local/nvidia/lib64:/usr/local/tcpx/lib64',
 ]
 
+NETWORK_STORAGE_TYPE = 'PERSISTENT'
+INSTANCE_STORAGE_TYPE = 'SCRATCH'
+INSTANCE_STORAGE_DISK_TYPE = 'local-ssd'
+INSTANCE_STORAGE_INTERFACE_TYPE = 'NVME'
+INSTANCE_STORAGE_DEVICE_NAME_PREFIX = '/dev/disk/by-id/google-local-nvme-ssd-'
+DEVICE_NAME_PREFIX = '/dev/disk/by-id/google-'
+
+BASH_SCRIPT_START = """#!/bin/bash
+    set -e
+    set -x
+"""
+DISK_MOUNT_USER_DATA_TEMPLATE = """
```

Comment:
same as above, do we need `textwrap.dedent`

Reply:
Updated